src/lib.rs

```diff
@@ -272,6 +275,8 @@ pub fn make_test_closure(config: &Config, testpaths: &TestPaths) -> test::TestFn
     let config = config.clone();
     let testpaths = testpaths.clone();
     test::DynTestFn(Box::new(move || {
+        #[cfg(feature = "norustc")]
+        let config = config.clone();  // FIXME: why is this needed?
```

Comment:
Maybe some of the NLL changes in rustc nightly obviates the need to clone `config`? Or we're looking at a bug in rustc nightly? `config` has type `&Config`, so it should be freely copyable across threads, right? It could also be a that's been fixed in nightly, but hasn't made its way to stable?

Reply:
It could be related to `test::DynTestFn` takes an `Box<FnMut() + Send>` instead of `Box<FnBox() + Send>`. (Because `FnBox` isn't stable)

Ref: https://github.com/servo/rustc-test/commit/aa605ed35527fcef027089accb10f63456298bfd